pandas/tseries/tests/test_base.py

```diff
@@ -2164,6 +2226,28 @@ def test_drop_duplicates_metadata(self):
         self.assert_index_equal(idx, result)
         self.assertEqual(idx.freq, result.freq)
 
+    def test_drop_duplicates(self):
+        # to check Index/Series compat
+        base = pd.period_range('2011-01-01', '2011-01-31', freq='D',
```

Comment:
no, the code directly above intends to check metadata. others look similar are for datetime/timedelta. though we have index/series separated tests, I thinks it's good chance to check index/series compat.